lib/index.js

```diff
@@ -3,7 +3,8 @@ Object.defineProperty(exports, "__esModule", { value: true });
 var path = require("path");
```

Comment:
This file is just the build output, in reality you would do this when publishing to NPM or with a Github action. But since we're referencing this package directly with Git we need to keep it up to date (the extra work is not worth the saved 30 seconds)